src/components/inputField/PasswordField/PasswordField.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import styles from './PasswordField.module.scss';
+import hide from '../../../assets/hidePasswordIcon.png';
+import show from '../../../assets/showPasswordIcon.png'; 
+
+/***
+ * Parameters for the password field
+ * header: Required. String of the header for the textbox. Example: "Enter Password"
+ * setInputValue: Required. Variable to set the input value of the password
+ */
+interface PasswordFieldProps {
+    header: string;
+    setInputValue: React.Dispatch<React.SetStateAction<string>>;
+}
```

Comment:
With TypeScript interfaces, you can add JSDoc comments to each key-value pair to strengthen your documentation. As an added bonus, VS Code will take those comments and display them whenever you hover over any of the interface's properties.

For example, you could modify this snippet as follows:
```suggestion
/**
 * Parameters for the password field
 */
interface PasswordFieldProps {
    /**
     * String of the header for the textbox.
     * @example
     * header={"Enter Password"}
     */
    header: string;
    /** Variable to set the input value of the password. */
    setInputValue: React.Dispatch<React.SetStateAction<string>>;
}
```

In doing so, you'll see the following suggestion modal whenever you hover over the `header` prop: 
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/assets/49412783/facc8d71-cb80-4457-87d2-a0fa1912e226)

You can keep the original comment for the interface as that also provides nice hover documentation:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/assets/49412783/32208495-d2dc-4727-a8cf-4d7b491e8183)